drush/README.md

```diff
@@ -1 +0,0 @@
-This directory contains commands, configuration and site aliases for Drush. See http://packages.drush.org/ for a directory of Drush commands installable via Composer.
```

Comment:
Unrelated change. Revert this. This file is part of the upstream drupal-project and we should not mess with it.